recipe-server/client/control_new/state/extensions/actions.js

```diff
@@ -74,3 +86,15 @@ export function updateExtension(pk, extensionData) {
     });
   };
 }
+
+export function saveExtensionListingColumns(columns) {
+  return async dispatch => {
+    await localForage.setItem('extension_listing_columns', columns);
```

Comment:
/me nods. i need to create a query component that pulls from localForage. I'll add that in.